From dc2f992f2ffa1a115f29efa6b5fedb0a7207f7c5 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Tue, 26 Feb 2019 16:53:34 +0100
Subject: [PATCH] pinctrl-equilibrium: Remove return value from void function

These functions are retuning void and the Linux API does not allow them to return anything, remove wrong return value.
This fixes the following compile warnings:
drivers/pinctrl/pinctrl-equilibrium.c: In function 'eqbr_pin_dbg_show':
drivers/pinctrl/pinctrl-equilibrium.c:646:10: warning: 'return' with a value, in function returning void
   return -ENODEV;
          ^
drivers/pinctrl/pinctrl-equilibrium.c: In function 'eqbr_pinconf_dbg_show':
drivers/pinctrl/pinctrl-equilibrium.c:985:10: warning: 'return' with a value, in function returning void
   return -ENODEV;
          ^
---
 drivers/pinctrl/pinctrl-equilibrium.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/pinctrl/pinctrl-equilibrium.c b/drivers/pinctrl/pinctrl-equilibrium.c
index adf92f6a2909..47d4a77f13d6 100644
--- a/drivers/pinctrl/pinctrl-equilibrium.c
+++ b/drivers/pinctrl/pinctrl-equilibrium.c
@@ -643,7 +643,7 @@ static void eqbr_pin_dbg_show(struct pinctrl_dev *pctldev, struct seq_file *s,
 	bank = find_pinbank_via_pin(pctl, pin);
 	if (!bank) {
 		dev_err(pctl->dev, "Couldn't find pin bank for pin %u\n", pin);
-		return -ENODEV;
+		return;
 	}
 	offset = pin - bank->pin_base;
 
@@ -982,7 +982,7 @@ static void eqbr_pinconf_dbg_show(struct pinctrl_dev *pctldev,
 	bank = find_pinbank_via_pin(pctl, pin);
 	if (!bank) {
 		dev_err(pctl->dev, "Couldn't find pin bank for pin %u\n", pin);
-		return -ENODEV;
+		return;
 	}
 	mem = bank->membase;
 	offset = pin - bank->pin_base;
